docs/building_on_fedora_34.md

```diff
@@ -0,0 +1,81 @@
+# Building on Fedora 34
+
+This documents step by step on how to compile and put into use the software `mod_tile` and `renderd`.
+Please see our [Continous Integration script](../.github/workflows/build-and-test-fedora-34.yml) for more detail.
+
+```shell
+#!/usr/bin/env bash
+
+# Update installed packages
+yum --assumeyes update
```

Comment:
Note that `sudo` will likely be required for most of the commands in this document (also in the document for `CentOS`). It is not currently used/needed since the default user from the relevant `Docker` images is `root`.

Reply:
Added sudos.